egs/librispeech/ASR/tdnn_lstm_ctc/asr_datamodule.py

```diff
@@ -0,0 +1,362 @@
+import argparse
```

Comment:
Not sure if it makes sense -- but maybe it's sufficient to have a single copy of this script one level of directories up, and if any recipe requires non-standard processing, it would make it's own copy at the "current" directory level?

Reply:
How about putting a symlink to other model directories to this file?
I was thinking that each model is as self-contained as possible.
If someone wants to modify this file, he/she can replace the symlink with a copy of this file.

Reply:
yeah that makes sense to me